src/gui/ogr/qgsnewogrconnection.h

```diff
@@ -52,6 +52,8 @@ class GUI_EXPORT QgsNewOgrConnection : public QDialog, private Ui::QgsNewOgrConn
 
   public slots:
     void accept() override;
+    //! Updates state of the OK button depending of the filled fields
```

Comment:
Is this public on purpose?
If yes, add a `\since`

Reply:
No, it should be private. Fixed now.

Thanks!